### DISCUSSION TITLE
Community meetings

### DISCUSSION THREAD

**Discussion by lizrice** [aquasecurity/starboard#106] [Announcements]
We have started holding community calls for the Aqua open source projects - all the details are in this 📝 [meeting document](https://hackmd.io/@aquasecurity/SyBkZoX2I). You're welcome to make suggestions for the agenda directly in that document. 

If you have any questions you can also ask them here!

**Comment by itaysk** [comment]
Updating that there are no further community calls. Please feel free to engage with us using [GitHub Discussions](https://github.com/aquasecurity/starboard/discussions), and if needed we'd be happy to schedule a call for topics that require collaboration beyond text.
